docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+   "execution_count": 12,
+   "id": "08446655",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['E', '##u', 'rejects', 'German', 'call', 'to', 'boycott', 'British', 'la', '##mb', '.']\n"
+     ]
+    }
+   ],
+   "source": [
+    "model = Ner(\"folds/fold0/model/\") \n",
+    "tokens = model.tokenize(sentences[i])[0] \n",
+    "print(tokens) \n",
+    "tokens = [token.replace('#', '') for token in tokens] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "493203d3",
+   "metadata": {},
+   "source": [
+    "`##` indicates that the token is a subword. We remove them and manually map them to the original tokens afterwards. We collect both the predictive probabilities and the tokens for each sentence. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "b9fac29e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 4144/4144 [01:14<00:00, 55.45it/s]\n",
+      "100%|██████████| 4144/4144 [01:15<00:00, 55.16it/s]\n",
+      "100%|██████████| 4144/4144 [01:14<00:00, 55.28it/s]\n",
+      "100%|██████████| 4143/4143 [01:14<00:00, 55.34it/s]\n",
+      "100%|██████████| 4143/4143 [01:14<00:00, 55.63it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "sentence_tokens, sentence_probs = {}, {} \n",
+    "for i in range(k): \n",
+    "    model = Ner(\"folds/fold%d/model/\" % i) \n",
+    "    for index in tqdm(indicies[i]): \n",
+    "        sentence_probs[index], sentence_tokens[index] = model.predict(sentences[index]) \n",
+    "        \n",
+    "sentence_tokens = [sentence_tokens[i] for i in range(len(sentences))] \n",
+    "sentence_probs = [sentence_probs[i] for i in range(len(sentences))] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "754a24f4",
+   "metadata": {},
+   "source": [
+    "Most tokenizers partition sentences into subword-level tokens without altering the characters. However, you should verify whether any characters are modified, particularly for edge cases such as single or double quotations. In this example, double quotations `\"` are broken down into two `'`s or two `` ` ``s. Run the following to verify if any of the characters in the sentences has been modified. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "237ff9c2",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(modified(given_words, sentence_tokens)) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b498649",
+   "metadata": {},
+   "source": [
+    "Given that some characters have been modified, we need to further map `sentence_tokens` back to the original characters. Note that such mapping is different for different models, and is usually not required by most tokenizers. "
```

Comment:
"we need to further map `sentence_tokens` back to the original characters"
-> 
"we need to map `sentence_tokens` back to the characters from the original dataset, so that ..." 

Here ... should clarify why this is useful to do and why we cannot just work with the new tokens